lib/src/material_color_picker.dart

```diff
@@ -41,11 +41,11 @@ class MaterialColorPicker extends StatefulWidget {
 class _MaterialColorPickerState extends State<MaterialColorPicker> {
   final _defaultValue = materialColors[0];
 
-  List<ColorSwatch> _colors = materialColors;
+  List<ColorSwatch>? _colors = materialColors;
 
-  ColorSwatch _mainColor;
-  Color _shadeColor;
-  bool _isMainSelection;
+  ColorSwatch? _mainColor;
+  Color? _shadeColor;
+  bool? _isMainSelection;
```

Comment:
You can initialize `_isMainSelection` to `false` and make it non-nullable, because all usages are `_isMainSelection ?? false` (i.e. all usages use `false` when it is not set). This means we can set it to `false` right away.